curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -11,7 +11,7 @@ dashedName: exercise-tracker
 Build a full stack JavaScript app that is functionally similar to this: <https://exercise-tracker.freecodecamp.rocks/>. Working on this project will involve you writing your code using one of the following methods:
 
 -   Clone [this GitHub repo](https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/) and complete your project locally.
--   Use [our repl.it starter project](https://repl.it/github/freeCodeCamp/boilerplate-project-exercisetracker) to complete your project.
+-   Use [our replit.com starter project](https://replit.com/github/freeCodeCamp/boilerplate-project-exercisetracker) to complete your project.
```

Comment:
```suggestion
-   Use [our Replit starter project](https://replit.com/github/freeCodeCamp/boilerplate-project-exercisetracker) to complete your project.
```

Reply:
That's a good catch. 

However, I see that the original issue does not mention this explicitly. It may make sense to audit all places instead of only the curriculum (as in this PR) and make the change.

Reply:
I agree that this can be merged as is and an additional PR could be created to change the name